Reference/Angular/Services/editorService/Index.md

```diff
@@ -0,0 +1,83 @@
+# Editor Service
+
+In Umbraco 8, the Angular `editorService` service is the primary resource used for opening overlays and handling infinite editing. Besides the `open` and `close` functions, the service also contains functions for opening specialized overlays/editors - eg. `contentPicker` or `mediaPicker`.
+
+### Content Picker
+
+The `contentPicker` function opens a content picker in infinite editing. Depending on the options, it may be used for picking a single content item or a set of content items. Options for the function is as following:
```

Comment:
```suggestion
The `contentPicker` function opens a Content Picker in infinite editing. Depending on the options, it may be used for picking a single content item or a set of content items. Options for the function is as following:
```